src/site/shared/utils/ImageExporter.ts

```diff
@@ -47,3 +47,39 @@ export function SavePDF(canvas: HTMLCanvasElement, projectName: string): void {
     pdf.addImage(data, "PNG", 0, 0, pdfWidth, pdfHeight);
     pdf.save(projectName + ".pdf");
 }
+
+
+export function SaveJPG(canvas: HTMLCanvasElement, projectName: string): void { 
```

Comment:
This and `SavePNG` and extremely similar, you should combine them into a single function called like `SaveImage` that has a 3rd parameter called like `type` which is `type: "png" | "jpg"` and then you can color the background if the type is `jpg`